source/tengine.js

```diff
@@ -637,6 +642,7 @@ tengine.ModelFileReader = class {
         this._graphs = [];
         const subgraphOffsets = reader.uint32s();
         for (const subgraphOffset of subgraphOffsets) {
+            subgraphOffsets.length =1 ;
```

Comment:
Why is this required?

Reply:
Hi Lutz, 

This is for Tengine's subgraph strategy. Tengine sets the main subgraph as subgraph[0], other subgraphs[1-n] are cut from subgraph[0]. These subgraphs may have different data type, due to a hybrid-precision strategy. For example, subgraph[1] is float32 while subgraph[2] uses uint8, they are both sub-graph to subgraph[0]. 

So basically, the subgraph[0] is not actually a 'subgraph'. It is 'the graph'. 
Subgraphs shouldn't be shown when rendering, only 'the graph' will be shown.

[hybrid-precision subgraph  model.zip](https://github.com/lutzroeder/netron/files/6471719/hybrid-precision.subgraph.model.zip)

We can use this model to test. It's also an experimental feature in Tengine.